concordia/models.py

```diff
@@ -234,6 +234,9 @@ def get_absolute_url(self):
             },
         )
 
+    def latest_transcription(self):
+        return self.transcription_set.order_by("-pk").first()
```

Comment:
I'm not opposed to having this as a class method but I'm thinking that for search we might want to have a prefetch to do this in bulk — something like `prefetch_related(latest_transcriptions=Prefetch('transcription_set', queryset=Transcription.objects.order_by('pk').latest())` & an accessor but I would want to test that.

Reply:
I like the idea of having it pre-fetch. I'm following the documentation for django-elasticsearch-dsl about "Using Different Attributes for Fields" https://pypi.org/project/django-elasticsearch-dsl/ 
If you have an alternative method to propose, I'm interested.

Reply:
Have you measured performance on this currently? I'm assuming it'll be slow enough to be a concern but we should probably confirm that before spending more time on it.

Reply:
No, I used it once a week ago, since then haven't looked at it.

Reply:
It's not super slow. I was able to fully rebuild all the indices in less than 15 minutes.

Reply:
Okay, let's table this until we have a problem worth attention